src/patternfly/components/AppLauncher/app-launcher-menu-item-external-icon.hbs

```diff
@@ -1,6 +1,13 @@
-<i class="pf-c-app-launcher__menu-item-external-icon fas fa-external-link-alt{{#if app-launcher-menu-item-external-icon--modifier}} {{app-launcher-menu-item-external-icon--modifier}}{{/if}}"
-  aria-hidden="true"
+<span class="pf-c-app-launcher__menu-item-external-icon
+  {{#if app-launcher-menu-item-external-icon--modifier}} 
+    {{app-launcher-menu-item-external-icon--modifier}}
+  {{/if}}
   {{#if app-launcher-menu-item-external-icon--attribute}}
```

Comment:
Since `app-launcher-menu-item-external-icon--attribute` is to generate html attributes, not extra strings to be passed to the `class` attribute, you'll need to add a close `"` between the conditional for `app-launcher-menu-item-external-icon--modifier` and the one for `app-launcher-menu-item-external-icon--attribute`, then remove the extra `"` below after the conditional for `app-launcher-menu-item-external-icon--attribute`.

Reply:
For reference, you can look at most of the other handlebars partials in this directory. The [app-launcher-menu-search.hbs](https://github.com/patternfly/patternfly/blob/master/src/patternfly/components/AppLauncher/app-launcher-menu-search.hbs) is a good one.